config/software/makedepend.rb

```diff
@@ -57,13 +56,10 @@
     }
   end
 
-configure_env["PKG_CONFIG_PATH"] = "#{install_dir}/embedded/lib/pkgconfig" +
+configure_env["PKG_CONFIG_LIBDIR"] = "#{install_dir}/embedded/lib/pkgconfig" +
   File::PATH_SEPARATOR +
   "#{install_dir}/embedded/share/pkgconfig"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Not sure I understand this part, do this mean the following ?
```
configure_env["PKG_CONFIG_LIBDIR"]="#{install_dir}/embedded/lib/pkgconfig/#{install_dir}/embedded/share/pkgconfig"
```

Reply:
`PATH_SEPARATOR` should be equal to `:` as it's a path *listing* separator, not a path separator. Naming is hard :sweat_smile: 
So it should amount to something like `PKG_CONFIG_LIBDIR=/path/to/embedded/lib/pkgconfig:/path/to/embedded/share/pkgconfig`

Reply:
🤯 Makes a lot of sense okkkk !